ui/lib/pki/addon/routes/certificates/index.js

```diff
@@ -4,18 +4,12 @@ import { inject as service } from '@ember/service';
 export default class PkiCertificatesIndexRoute extends Route {
   @service store;
   @service secretMountPath;
-  @service pathHelp;
-
-  beforeModel() {
```

Comment:
is this no longer a think because we're querying the base certificate!? 🤩

Reply:
No I just couldn't get openAPI to work for the designs so I manually defined the attributes on the base certificate model instead. The `pathHelp` service basically instantiates a new model with the attrs from the openAPI spec so we don't need that here anymore.